lib/realtime_web/router.ex

```diff
@@ -104,6 +104,7 @@ defmodule RealtimeWeb.Router do
     pipe_through([:open_cors, :tenant_api, :secure_tenant_api])
 
     post("/broadcast", BroadcastController, :broadcast)
+    delete("/disconnect_clients", TenantController, :disconnect_clients)
```

Comment:
Should we have only a `/suspend` and `/unsuspend` endpoint where it disconnects clients and kills all the db connections including extensions? 

I was having trouble this week figuring out how to restart the `realtime_subscription_manager_pub` connections. 

Having one endpoint where we could confidently disable Realtime for all the db and client conns for a tenant is needed.

Reply:
that's a really good , will implement that instead